VERSION

```diff
@@ -1 +1 @@
-v0.37.0
+v0.40.0
```

Comment:
I'm curious, why did you skip 2 versions?

Reply:
> I'm curious, why did you skip 2 versions?

In case we ended up needing to update the previous style of SDK for a point release branch or something.